docs/04-guides/get-viewer-engagement/02-api.mdx

```diff
@@ -0,0 +1,114 @@
+---
+sidebar_class_name: tabbed-page
+hide_title: true
+slug: api
+title: API
+pagination_next: guides/playback-a-video-stream
+---
+
+import Tabs from "./_tabs.mdx";
+import CodeBlock from "@theme/CodeBlock";
+import CodeBlockTabs from "@site/src/components/CodeBlockTabs";
+import CodeBlockTabItem from "@site/src/components/CodeBlockTabItem";
+
+<Tabs />
+
+The Livepeer API allows you to send video files to Livepeer and get them ready
+for optimized playback. Videos can be provided either by you (static content) or
+your users, given your application offers an interface for them to do so.
+
+## Step 1: Get the `asset.id` of an existing asset
+
+Get the `asset.id` of an existing asset. An `asset.id` can be found in the
+response object of any API call working with assets. If you haven't created an
+asset yet, you can follow the
+[upload a video asset guide](/guides/on-demand/upload-video-asset#api) to do so.
+
+## Step 2: Retrieve viewership data
+
+Once you have the `asset.id`, you can make a request to get the viewership data.
+
+<CodeBlockTabs>
+<CodeBlockTabItem value="JavaScript" label="JavaScript">
+
+```js
+const getViews = await fetch("https://livepeer.studio/api/data/views/{assetId}/total", {
+    method: "GET",
+    headers: {
+        Authorization: `Bearer ${process.env.API_TOKEN}`,
+    },
+});
+```
+
+</CodeBlockTabItem>
+<CodeBlockTabItem value="curl" label="curl">
+
+```bash
+curl --location --request GET https://livepeer.studio/api/data/views/{assetId}/total \
+    -H "Authorization: $LP_API_KEY"
+```
+
+</CodeBlockTabItem>
+</CodeBlockTabs>
+
+## Livepeer player alternative
```

Comment:
We should remove this whole section for now since we don't support alternative players.

Also sent a suggestion on how to phrase the configuring a player section to make this clear.